config/routes.rb

```diff
@@ -166,7 +166,11 @@
           get :tooltip_locked
         end
       end
-      resources :approaches, only: %i[index show], controller: "tracks/approaches" do
+
+      resources :articles, only: %i[index show], controller: "tracks/articles"
+      resources :approaches, only: %i[index show], controller: "tracks/approaches"
+
+      resources :dig_deeper, only: %i[index], controller: "tracks/dig_deeper" do
```

Comment:
```suggestion
      resource :dig_deeper, only: %i[index], controller: "tracks/dig_deeper" do
```

This should be a single resource (as there can't be multiple dig-deepers as the resource name is singular) and the action should be `show` not index. You can then remove the `collection do` line below this too.

Reply:
Neat!